app/src/main/java/com/android/aniviewer_kun/MainViewModel.kt

```diff
@@ -23,24 +23,91 @@ class MainViewModel: ViewModel() {
     var fetchDone : MutableLiveData<Boolean> = MutableLiveData(false)
 
     private var media = MutableLiveData<List<MediaListQuery.Medium?>?>()
+    private var searchMediaResults = MutableLiveData<List<MediaListQuery.Medium?>?>()

```

Comment:
saved search results independently as it's a short list and having to retype on return could be annoying